gulpfile.js

```diff
@@ -1,14 +1,34 @@
 var copy = require("copy"),
     clear = require("cli-clear"),
     del = require("del"),
+    exec = require("child_process").exec,
     gulp = require("gulp"),
-    gutil = require("gulp-util");
+    gutil = require("gulp-util"),
+    runSequence = require("run-sequence");
 
 gulp.task("clear", function(cb) {
     clear();
     cb();
 });
 
+gulp.task("clean", function(cb) {
+    del("./coverage")
+    .then(function(files) {
+        if(files.length > 0) {
+            gutil.log("Deleted %s", files);
+        }
+        cb();
```

Comment:
If you return the promise within the task, you don't need to use the gulp `cb` :)

Reply:
Done!